Mathlib/Analysis/Analytic/Basic.lean

```diff
@@ -588,6 +595,17 @@ theorem AnalyticOn.mono {s t : Set E} (hf : AnalyticOn 𝕜 f t) (hst : s ⊆ t)
   fun z hz => hf z (hst hz)
 #align analytic_on.mono AnalyticOn.mono
 
+theorem AnalyticOn.congr' {s : Set E} (hf : AnalyticOn 𝕜 f s) (hg : ∀ z ∈ s, f =ᶠ[𝓝 z] g) :
```

Comment:
Is the following weaker or stronger?
```suggestion
theorem AnalyticOn.congr' {s : Set E} (hf : AnalyticOn 𝕜 f s) (hg : f =ᶠ[𝓝ˢ s] g) :
```

Reply:
They're equivalent, I believe, by `mem_nhdsSet_iff_forall`.

https://github.com/leanprover-community/mathlib4/blob/2e06bd9f0f56ec7f72fad9f6bc0acd8ccdc0bf13/Mathlib/Topology/NhdsSet.lean#L47

Reply:
In that case, I suspect the spelling I suggested is preferable?

Reply:
I agree! Just pushed the change.